package.json

```diff
@@ -13,6 +13,8 @@
         "dotenv": "^10.0.0",
         "express": "~4.16.1",
         "http-errors": "~1.6.3",
+        "mongodb": "^3.6.9",
```

Comment:
Not sure if we need to declare dependency for `mongodb` here when we already have `mongoose`.